### DISCUSSION TITLE
Button order

### DISCUSSION THREAD

**Discussion by DigitaliusPL** [orestbida/cookieconsent#469] [Q&A] [answered]
How to edit button order? I would like the "Accept" button to be first (left) and the "Reject" button second. In both modals...

I would like to see this change of button location also in "/demo/demo-gtm"

Talking about: cookieconsent v3.0.0-rc.13

**Comment by orestbida** [comment, marked as answer]
You can set `flipButtons: true` via [`guiOptions`](https://cookieconsent.orestbida.com/reference/configuration-reference.html#guioptions-consentmodal).
